_docs/_latest/configure/forseti-visualizer/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Forseti Visualizer
```

Comment:
See comment below on not having this page.

Reply:
Done